tests/sentry/tasks/test_post_process.py

```diff
@@ -1309,7 +1310,7 @@ def create_event(self, data, project_id, assert_no_errors=True):
             data=data, project_id=project_id, assert_no_errors=assert_no_errors
         )
 
-        occurrence_data = self.build_occurrence_data(event_id=event.event_id, project_id=project_id)
+        occurrence_data = self.build_occurrence_data(event_id=event.event_id)
```

Comment:
AH DAMN!  Found my test bug, unsure how I accidentally deleted this part.  Fix coming.  @wedamija 
Confident my changes do not functionally change much, but a lot of test adjustments led to some fat fingers.

Reply:
I think that this was probably a merge issue, we added `project_id` as a param here recently